### ISSUE TITLE
SlackRTM can't import slackClient, wich is installed

### ISSUE THREAD

**Issue by 00svbeon** [hangoutsbot/hangoutsbot#456]
```
2015-12-28 16:47:27 DEBUG plugins: slackrtm matched to slackrtm

2015-12-28 16:47:27 ERROR plugins.slackrtm: missing module slackclient: pip3 install --upgrade python-slackclient >=0.16

2015-12-28 16:47:27 ERROR plugins: EXCEPTION during plugin import: plugins.slackrtm
Traceback (most recent call last):
  File "/home/slack/h/hangupsbot/plugins/__init__.py", line 327, in load
    importlib.import_module(module_path)
  File "/usr/lib/python3.4/importlib/__init__.py", line 109, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 2231, in _gcd_import
  File "<frozen importlib._bootstrap>", line 2214, in _find_and_load
  File "<frozen importlib._bootstrap>", line 2203, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 1200, in _load_unlocked
  File "<frozen importlib._bootstrap>", line 1129, in _exec
  File "<frozen importlib._bootstrap>", line 1448, in exec_module
  File "<frozen importlib._bootstrap>", line 321, in _call_with_frames_removed
  File "/home/slack/h/hangupsbot/plugins/slackrtm.py", line 60, in <module>
    from slackclient import SlackClient
  File "/usr/local/lib/python3.4/dist-packages/slackclient/__init__.py", line 1, in <module>
    from slackclient._client import SlackClient
  File "/usr/local/lib/python3.4/dist-packages/slackclient/_client.py", line 6, in <module>
    from slackclient._server import Server
  File "/usr/local/lib/python3.4/dist-packages/slackclient/_server.py", line 7, in <module>
    from websocket import create_connection
ImportError: cannot import name 'create_connection'
2015-12-28 16:47:27 INFO root: bot initialised
```

**Comment by ppescitelli** [comment]
I used pip install slackclient to get the bot to compile. Just need to finish the slack integration.

**Comment by 00svbeon** [comment]
I hope pip3? Well i've done that as well, but it isnt't working still...

lör 2 jan 2016 16:54 ppescitelli <notifications@github.com> skrev:

> I used pip install slackclient to get the bot to compile. Just need to
> finish the slack integration.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hangoutsbot/hangoutsbot/issues/456#issuecomment-168401707>
> .
>
-- 

//S

**Comment by pleasantone** [comment]
Fixed in #465
